drivers/bluetooth/hci/Kconfig

```diff
@@ -56,6 +56,18 @@ config BT_STM32_IPM
 	help
 	  TODO
 
+config BT_SILABS_HCI
+	bool "Silicon Labs Bluetooth interface"
+	depends on SOC_SERIES_EFR32BG22
+	select ENTROPY_GENERATOR
+	select MBEDTLS
+	select MBEDTLS_PSA_CRYPTO_C
+	select MBEDTLS_ENTROPY_ENABLED
+	select MBEDTLS_ZEPHYR_ENTROPY
+	help
+	  Use Silicon Labs binary Bluetooth library to connect to the
```

Comment:
It seems like using this component like this:

```
$ west build --board efr32bg_sltb010a zephyr/samples/bluetooth/observer
```

does not print a warning about the build being tainted. I think [it should](https://docs.zephyrproject.org/latest/contribute/bin_blobs.html#tainting).
```
$ grep TAINT build/zephyr/.config
```

Reply:
Hi, thanks for commenting. I'm not sure what additional Kconfig changes should I make here, as I couldn't find a preexisting BT driver that does that. I've tested with `esp32c3_devkitm`, which uses blobs as well, and no warning was printed by the build system.

As for taint marking in the Kconfig itself, we're already doing that:
```
% west build -p -b efr32bg_sltb010a samples/bluetooth/observer >
% grep TAINT build/zephyr/.config
CONFIG_TAINT_BLOBS=y
CONFIG_TAINT=y
```

Reply:
> Hi, thanks for commenting. I'm not sure what additional Kconfig changes should I make here, as I couldn't find a preexisting BT driver that does that. I've tested with `esp32c3_devkitm`, which uses blobs as well, and no warning was printed by the build system.

True. Guess fixing this does not have to be part of this PR then.

> As for taint marking in the Kconfig itself, we're already doing that:
> 
> ```
> % west build -p -b efr32bg_sltb010a samples/bluetooth/observer
> % grep TAINT build/zephyr/.config
> CONFIG_TAINT_BLOBS=y
> CONFIG_TAINT=y
> ```
This works when the espressif blobs got fetched, but not when only the silabs ones are.

Try this again after executing the following:
```shell
$ west blobs clean
$ west blobs fetch silabs
```
It will result in a build which does not have TAINT_BLOBS set.

This behaviour seems to boil down to https://github.com/zephyrproject-rtos/zephyr/blob/3d5de8920b971dc3fcb3b2a1504e28737396d25e/scripts/zephyr_module.py#L332 only selecting TAINT_BLOBS when blobs have been found AND `zephyr/Kconfig` exists in the modules. The file can be empty as it is in [the espressif HAL](https://github.com/zephyrproject-rtos/hal_espressif/blob/823e1df5f5b5ae0254f046f6e255cdd7a84a9095/zephyr/Kconfig).

Not sure if changing zephyr_module.py or adding zephyr/Kconfig to the silabs HAL module is the better solution.

Reply:
Ok, thanks for elaborating on that. I'll investigate it.

Reply:
@rettichschnidi We opted for the latter. Could you retest it and confirm that it works as well on your end?

Reply:
It does. Thanks!